prototype2/common/HistSerializer.h

```diff
@@ -11,23 +11,24 @@
 
 #include "mo01_nmx_generated.h"
 
-#include <common/Producer.h>
 #include <common/Hists.h>
-#include <libs/include/gccintel.h>
+#include <common/Producer.h>
 
 class HistSerializer {
 public:
-  /// \todo document
-  HistSerializer();
+  /** \todo document */
```

Comment:
Naming the class "HistogramSerializer" makes it slightly more clear what it does.